From 83405f6a1e75e8efe23989b58e2d9e8b09af43ef Mon Sep 17 00:00:00 2001
From: Islam Amer <islam.amer@jollamobile.com>
Date: Tue, 29 Apr 2014 23:44:09 +0300
Subject: [PATCH] [tests] use new twisted gireactor to work with gi based
 pygobject

---
 tests/twisted/servicetest.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/twisted/servicetest.py b/tests/twisted/servicetest.py
index 4dc604f..1b8597d 100644
--- a/tests/twisted/servicetest.py
+++ b/tests/twisted/servicetest.py
@@ -20,9 +20,9 @@
 Infrastructure code for testing Mission Control
 """
 
-from twisted.internet import glib2reactor
+from twisted.internet import gireactor
 from twisted.internet.protocol import Protocol, Factory, ClientFactory
-glib2reactor.install()
+gireactor.install()
 import sys
 
 import pprint
-- 
1.9.0

