diff -Naur telepathy-mission-control-5.14.0/configure.ac telepathy-mission-control-5.14.0-test-dir-fix/configure.ac
--- telepathy-mission-control-5.14.0/configure.ac	2012-10-03 17:08:47.000000000 +0000
+++ telepathy-mission-control-5.14.0-test-dir-fix/configure.ac	2013-04-03 09:50:20.434116144 +0000
@@ -129,7 +129,7 @@
 fi
 AM_CONDITIONAL([ENABLE_INSTALLED_TESTS], [test "x$installed_tests" = xyes])
 
-mctestsdir=${libdir}/telepathy-mission-control-tests
+mctestsdir=/opt/tests/telepathy-mission-control
 AC_SUBST(mctestsdir)
 
 AC_ARG_WITH([accounts_dir], [AS_HELP_STRING([--with-accounts-dir=<path>],[Directory in which accounts were previously stored])])
