diff -Naur telepathy-mission-control-5.15.0/tests/twisted/tools/run-mc.sh.in telepathy-mission-control-5.15.0-fix-system-bus-for-tests/tests/twisted/tools/run-mc.sh.in
--- telepathy-mission-control-5.15.0/tests/twisted/tools/run-mc.sh.in	2012-05-07 18:02:03.000000000 +0000
+++ telepathy-mission-control-5.15.0-fix-system-bus-for-tests/tests/twisted/tools/run-mc.sh.in	2013-05-07 09:23:12.759608336 +0000
@@ -2,4 +2,6 @@
 
 ulimit -c unlimited
 
+export DBUS_SYSTEM_BUS_ADDRESS=$DBUS_SESSION_BUS_ADDRESS
+
 @libexecdir@/mission-control-5
